engine/tester/config_parse.c

```diff
@@ -1423,7 +1540,8 @@ alloc_and_get_var_arg(xmlNodePtr node, bool is_var,
     p->variable = is_var;
     p->global = false;
     TAILQ_INIT(&p->values.head);
-    TAILQ_INSERT_TAIL(list, p, links);
+    if (is_var)
```

Comment:
The condition looks strange. Let's move addition to the list in the case of variable as well and use helper function in any case.